source/_includes/custom/navigation.html

```diff
@@ -9,7 +9,7 @@
       </a>
       <div class="nav-collapse collapse">
         <ul class="nav">
-          <li {% if page.url == '/index.html' %}class="selected"{% endif %}>
+          <li {% if page.url == '/index.html' or page.url contains '/blog/'%}class="selected"{% endif %}>
```

Comment:
When clicking on the Archives section, both Blog and Archives are showing as selected because of this new conditional.  Update this conditional statement to ignore the instance when the sub section is "archives" under blog.